scripts/jenkins/Jenkinsfile

```diff
@@ -741,36 +741,64 @@ def buildDoc() {
           [parserName: 'Doxygen', pattern: 'build/doc/doxygen.log']
         ]
 
-        // TODO don't write to latest on PR's
-        sshPublisher(
-          publishers: [
-            sshPublisherDesc(
-              configName: 'doc.libelektra.org',
-              transfers: [
-                sshTransfer(
-                  sourceFiles: 'build/doc/latex/*',
-                  removePrefix: 'build/doc/',
-                  remoteDirectory: 'api/latest'
-                ),
-                sshTransfer(
-                  sourceFiles: 'build/doc/man/*',
-                  removePrefix: 'build/doc/',
-                  remoteDirectory: 'api/latest'
-                ),
-                sshTransfer(
-                  sourceFiles: 'doc/help/*.html',
-                  removePrefix: 'doc/help/',
-                  remoteDirectory: 'help'
-                ),
-                sshTransfer(
-                  sourceFiles: 'build/API_DOC/*',
-                  removePrefix: 'build/API_DOC/'
-                )
-              ]
-            )
-          ],
-          verbose: true
-        )
+        if (isMaster()) {
+          sshPublisher(
+            publishers: [
+              sshPublisherDesc(
+                configName: 'doc.libelektra.org',
+                transfers: [
+                  sshTransfer(
+                    sourceFiles: 'build/doc/latex/*',
+                    removePrefix: 'build/doc/',
+                    remoteDirectory: 'api/latest'
+                  ),
+                  sshTransfer(
+                    sourceFiles: 'build/doc/html/*',
+                    removePrefix: 'build/doc/',
+                    remoteDirectory: 'api/latest'
+                  ),
+                  sshTransfer(
+                    sourceFiles: 'build/doc/man/man3elektra/*',
+                    removePrefix: 'build/doc/man/man3elektra/',
+                    remoteDirectory: 'api/latest/man'
+                  ),
+                  // TODO: API doc not built anymore
```

Comment:
Please add information that this is the API documentation for Elektra’s REST service to this comment. Otherwise someone might assume this folder contains documentation for Elektra’s C/C++ API.

Reply:
Done!